CMSgov/mint-app
# Easi 4484/ch backend cleanup v2

# EASI-4484
Address remaining TODO notes from the change history branch.
## Description

- Generically provide mechanism to skip translating table by table ID.
- Update some todos to Future Enhancement notes that were not needed for deployment
- Stop fetching the model plan directly for the audit, since the data was not need.
- Merge in Main (note, most of these file changes are actually just from merging in main)
- Simplify Seed command to only have one translate command
- Unit test to validate that db tags are not duplicated on a translation.



## How to test this change
1. Functionality should remain unchanged.
2. Please do a smoke test on change history, and also see if there are any addressed comments you don't agree with.


## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
